fastlane/metadata/default/subtitle.txt

```diff
@@ -1 +1 @@
-Build a website or blog
\ No newline at end of file
+Build a website or blog
```

Comment:
Had to highlight the diff to understand what saw broken

![image](https://user-images.githubusercontent.com/1218433/121829045-cb576e00-cd04-11eb-942b-3b4993f6a5e8.png)


🤯 🔍